src/django/api/gazetteer.py

```diff
@@ -179,7 +179,12 @@ def build_index_table(self, canonical):
 
         with connection.cursor() as cursor:
             cursor.execute(
-                "CREATE TABLE {} (LIKE dedupe_indexed_records INCLUDING ALL)"
+                """CREATE TABLE IF NOT EXISTS {}
+                (block_key text,
+                record_id varchar(32),
+                record_data text,
+                UNIQUE (block_key, record_id))
+                """
                 .format(table_name)
```

Comment:
Starting with the `dedupe_indexed_records` table created by the Django model

```
openapparelregistry=# \d dedupe_indexed_records
                                      Table "public.dedupe_indexed_records"
   Column    |         Type          | Collation | Nullable |                      Default
-------------+-----------------------+-----------+----------+----------------------------------------------------
 id          | integer               |           | not null | nextval('dedupe_indexed_records_id_seq'::regclass)
 block_key   | text                  |           | not null |
 record_id   | character varying(32) |           | not null |
 record_data | text                  |           | not null |
Indexes:
    "dedupe_indexed_records_pkey" PRIMARY KEY, btree (id)
    "dedupe_indexed_records_idx" btree (block_key, record_id)
    "unique_blockkey_recordid" UNIQUE CONSTRAINT, btree (block_key, record_id)

```

And comparing it to the table generated by 

```sql
CREATE TABLE IF NOT EXISTS foo
                (block_key text,
                record_id varchar(32),
                record_data text,
                UNIQUE (block_key, record_id))

```

We see several diferences

```
openapparelregistry=# \d foo
                          Table "public.foo"
   Column    |         Type          | Collation | Nullable | Default
-------------+-----------------------+-----------+----------+---------
 block_key   | text                  |           |          |
 record_id   | character varying(32) |           |          |
 record_data | text                  |           |          |
Indexes:
    "foo_block_key_record_id_key" UNIQUE CONSTRAINT, btree (block_key, record_id)
```

This version creates a table that looks more like the Django original

```sql
CREATE TABLE IF NOT EXISTS bar
                (id SERIAL PRIMARY KEY,
                block_key text NOT NULL,
                record_id varchar(32) NOT NULL,
                record_data text NOT NULL,
                UNIQUE (block_key, record_id));
CREATE INDEX bar_idx ON bar (block_key, record_id);
```

```
openapparelregistry=# \d bar
                                      Table "public.bar"
   Column    |         Type          | Collation | Nullable |             Default
-------------+-----------------------+-----------+----------+---------------------------------
 id          | integer               |           | not null | nextval('bar_id_seq'::regclass)
 block_key   | text                  |           | not null |
 record_id   | character varying(32) |           | not null |
 record_data | text                  |           | not null |
Indexes:
    "bar_pkey" PRIMARY KEY, btree (id)
    "bar_block_key_record_id_key" UNIQUE CONSTRAINT, btree (block_key, record_id)
    "bar_idx" btree (block_key, record_id)
```

Here is a suggested code change to implement this

```suggestion
                """
                CREATE TABLE IF NOT EXISTS {table_name}
                    (id SERIAL PRIMARY KEY,
                    block_key text NOT NULL,
                    record_id varchar(32) NOT NULL,
                    record_data text NOT NULL,
                    UNIQUE (block_key, record_id));
                CREATE INDEX {table_name}_idx
                ON {table_name} (block_key, record_id);
                """
                .format(table_name=table_name)
```